backend/src/entities/partitions/service.py

```diff
@@ -52,6 +52,27 @@ def get_partition_by_id(conn: Connection, partition_id: UUID) -> Partition:
     return _parse_row(result)
 
 
+def create_partition_tags(conn: Connection, partition: PartitionCreate, result) -> None:
```

Comment:
```suggestion
def create_partition_tags(conn: Connection, partition: PartitionCreate, masterclass_id: UUID | str) -> None:
```
Bosser avec des `sa.row`c'est un peux chiant pour le typing, comme conseil soit tu peux directement passer l'id si t'as pas besoin de recuperer d'autres attributs de la row
Tu peux aussi demander un object de classe Partion directement:
```suggestion
def create_partition_tags(conn: Connection, partition: Partition) -> None:
```
Et dans ton create faire 
```py 
result = ...
partition = _parse_row(result)
create_partion_tags(conn, partition)
return partition
```